src/crimson/net/ProtocolV2.h

```diff
@@ -71,6 +76,26 @@ class ProtocolV2 final : public Protocol {
   uint64_t peer_global_seq = 0;
   uint64_t connect_seq = 0;
 
+  seastar::future<> execution_done = seastar::now();
+
+  class Timer {
+    double last_dur_ = 0.0;
```

Comment:
could you s/last_dur_/last_duration/ or just s/last_dur/duration/ ?

we don't use underscore as the postfix of member variable. and member variable might need a better name as it's a way to document the interface.